### PR TITLE
[WIP] Optimize for ECMAScript 6.

### PR DESCRIPTION
**Pull request by kepler0**

### CODE REVIEW

File: lib/google-analytics.js [hunk 1]
```diff
@@ -52,30 +44,35 @@ module.exports = (function () {
 
     GoogleAnalytics.getCid(function (cid) {
       _.extend(params, { cid: cid }, GoogleAnalytics.defaultParams());
-      GoogleAnalytics.request('https://www.google-analytics.com/collect?' + querystring.stringify(params));
+      this.request('https://www.google-analytics.com/collect?' + querystring.stringify(params));
```
Comment (noseglid) [inline lib/google-analytics.js]: `this` is not defined here. You'd need to change the function definition to `(cid) => {`.

Comment (kepler0) [inline lib/google-analytics.js]: Good catch. Thanks. I've been using the unit tests to make sure that nothing breaks. If it's not covered by unit tests, I won't catch it.

**Comment by steelbrain** [comment]
While you are doing this, may I suggest [etch](https://github.com/nathansobo/etch) instead of space-pen-views?

**Comment by kepler0** [comment]
@steelbrain Different Issue, maybe? I just want to patch this so it is using ECMAScript 2015 classes.

**Comment by steelbrain** [comment]
You have my :+1:

**Comment by kepler0** [comment]
woot

File: lib/google-analytics.js (hunk 1 above)
Comment (noseglid) [inline lib/google-analytics.js]: Google analytics are left out of specs as of now. Not sure if it make sense to test it or not. If it breaks it wont affect user experience anyway

Comment (kepler0) [inline lib/google-analytics.js]: I'll end up testing this in my own environment when I'm done, so any glaring issues should be taken care of then.

**Comment by noseglid** [comment]
This is finalized in #229

**Comment by kepler0** [comment]
Awesome!

**Pull request closed without merging**